serverless-fraud-detection/docs/modules/ROOT/pages/serverless-fraud-detection.adoc

```diff
@@ -0,0 +1,555 @@
+= Serverless Fraud Detection with AWS and {hazelcast-cloud}
+:description: In this tutorial, you'll build a serverless solution for detecting fraudulent transactions using AWS Lambda, Hazelcast Viridian, and Node.js.
+:page-layout: tutorial
+:page-product: cloud
+:page-categories: Caching, AWS Lambda
+:page-est-time: 30 mins
+:page-lang: node
+:url-code-sample: https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/serverless-fraud-detection
+
+{description}
+
+== Context
+
+Every time your credit or debit card initiates a transaction, the underlying payment system validates the transaction with a series of checks. Some checks are simple, such as making sure that you have enough funds. Other checks are more advanced, such as filtering the history of card payments to identify a personalized pattern and validating the given transaction against that pattern.
+
+In this tutorial, you'll build an application that validates transactions made at airports, using the following process:
+
+. Compare the time between two transactions. A is the previous transaction and B is the current transaction.
+. Calculate the distance between the airports at which the transactions were made, using the link:https://en.wikipedia.org/wiki/Haversine_formula[haversine formula].
+. Determine whether someone could move from one airport to the other within the given timeframe.
+
+For example, if transaction A is made in London and transaction B is made in Paris, and the time between them is greater than two hours, then the transaction is *valid*.
+
+If transaction A is made in London and transaction B is made in Sydney, and the time between them is less than two hours, then the transaction is *suspicious*.
+
+Every time a new transaction is made, the airport coordinates are read and used as input in the haversine formula. If the data is slow to read, the whole process takes longer, which delays the transaction. To make sure that this data is always available and fast to read, it is cached in {hazelcast-cloud} because it offers fast-access in-memory storage.
+
+=== Architecture
+
+The whole stack is co-located in the same AWS region to ensure the shortest network paths across the components.
+
+image:serverless-fraud-detection:architecture.png[A banking client sends POST request to the serverless fraud detection application]
+
+- The S3 bucket stores the airport data.
+
+- The `ImportAirportsFn` Lambda function ingests the airport data into the *{hazelcast-cloud} cluster* so that it can be *queried with the lowest latency*. This function is triggered by uploads to the S3 bucket so that any new data is ingested to the {hazelcast-cloud} cluster.
```

Comment:
```suggestion
- The `ImportAirportsFn` Lambda function ingests the airport data into the {hazelcast-cloud} cluster so that it can be queried with the lowest latency. This function is triggered by uploads to the S3 bucket so that any new data is ingested to the {hazelcast-cloud} cluster.
```